Assets/_BForBoss/_Shaders/FullScreenShaders/HitEffectDemo.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering.HighDefinition;
+
+namespace BForBoss
+{
+    public class HitEffectDemo : MonoBehaviour
+    {
+        private Material mat;
+        private CustomPassVolume cps;
```

Comment:
So are all Post Processing Volume in HDRP `CustomPassVolume`?. 

So even like slow motion/dash volumes would be `CustomPassVolume` instead of `Volume` like we had in URP?

Reply:
Basically yeah. Any fullscreen effect is achieved using a CustomPassVolume in HDRP.

Reply:
We have a `PostProcessingVolumeWeightTool`, can you see if you can refactor this to make it fit this `CustomPassVolume` usecase?

Reply:
So URP post processing normally goes between 0 - 1, idk if it'll fit 1:1, but if u want to make something similiar, that could work too.

Reply:
If it's too much work/out of scope, it's fine. But you should put this script into `Utility`